distribution/docs/src/main/resources/_contents/_resources/content-directory-monitor-contents.adoc

```diff
@@ -72,6 +72,12 @@ To set multi-valued attributes, use a separate override for each value. For exam
 `*topic.keyword=PPI*` +
 `*topic.keyword=radiology*`
 
+.Blacklist
+The CDM blacklist uses the "bad.files" and "bad.file.extensions" properties from the system.properties file in "etc/" in order to prevent
+malicious or unwanted data from being ingested into DDF.  While the CDM will automatically omit hidden files, this is particularly useful when
+an operating system automatically generated files that should not be ingested.  One such example of this is "thumbs.db" in Windows.  This file type, as well as
+temporary files are included in the blacklist.
```

Comment:
Oxford comma needed: `...This file type, as well as temporary files, are included in the blacklist. `

*OR*

Re-reword and get rid of commas completely (better): `This file type and any temporary files are included in the blacklist.` (as long as there does *not* exist temp files that *can* be ingested)